dashboard/config/marketing/announcements.json

```diff
@@ -1,6 +1,6 @@
 {
   "pages": {
-    "/home": "access-report-launch-2024",
+    "/home": "gen-ai-launch-2024",
```

Comment:
Do we want to just remove the access report config below at this point?

Reply:
Sure thing!